packages/grafbase-nextjs-plugin/package.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
should the folder be nextjs-plugin or grafbase-nextjs-plugin?

Reply:
File naming can be anything, but it's recommended to use prefix to identify public packages from internal ones

Reply:
What does internal ones mean?

Reply:
packages with 'grafbase-' prefix are published to npm, the rest like eslint ts configs, are using only inside workspace